server/impl/render-light-page.js

```diff
@@ -0,0 +1,14 @@
+const request = require("request");
+
+const renderLightPage = (req, res, result) => {
```

Comment:
Render light page should accept config / client and return content. Please don’t set headers elsewhere and no body. On top of that if this errors out, then the node connection is never terminated